src/include/ipc/topology.h

```diff
@@ -47,6 +47,7 @@ enum sof_comp_type {
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
 	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
+	SOF_COMP_DRC,
```

Comment:
i don't think we need this since we are using uuid or is this just here for testbench sake?

Reply:
Yes, it is temp for testbench sake and should be replaced by uuid usage